### ISSUE TITLE
Question about loss_seg_src in SStrain.py?

### ISSUE THREAD

**Issue by KaiiZhang** [YanchaoYang/FDA#18]
Thanks a lot for your excellent work!
I found the loss_seg_src are used in SStrain.py. But in other related work, the model is usually only fine-tuned training on pseudo-label samples. I would like to ask if there is a big difference between these two methods
